skeleton-tests/test_addon_all.py

```diff
@@ -282,7 +284,30 @@ def test_addon_all(tmpdir, capsys, config):
     )
     assert result == 0
 
-    assert file_exists(wd, "/src/collective/task/configure.zcml")
+    assert file_exists(wd, "/src/collective/task/views/configure.zcml")
+
+    # generate subtemplate form:
+    template = """[variables]
+view_python_class_name=MyCoolForm
+view_name=my-cool-form
+view_register_for=Folder
```

Comment:
@MrTango to make tests working, you need to add `subtemplate_warning=True` in answers' template here, or to commit changes. (this is the mrbob's question  you are missing to reply 'Please commit your changes, before using a sub-template! Continue anyway? [N/y]').